packages/binding-modbus/src/modbus-connection.ts

```diff
@@ -461,12 +483,21 @@ export class PropertyOperation {
     done(base?: number, buffer?: Buffer): void {
         debug("Operation done");
 
+        if (!this.resolve || !this.reject) {
+            throw new Error("Invoked done before executing operation");
```

Comment:
```suggestion
            throw new Error("Invoke done before executing operation");
```

I think that's what you meant, right?
OR

`Invoke called before executing operation`

Reply:
Mmm not sure, what I meant is that the function `done` was invoked before executing the operation. Maybe it is better to be more explicit here: `Invoked function "done" before executing the Modbus operation` ?

Reply:
Your proposal makes sense.
An alternative *could* be 

`Function "done" was executed before the (this?) Modbus operation`

Either way is fine by me 👍

Reply:
I changed it a little bit more in https://github.com/eclipse-thingweb/node-wot/pull/1050/commits/dbf99795f515e01401a5d17ab1a460046cb2f0e3. I hope it is ok, merge if it is ok! 🛩️